packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -396,6 +396,17 @@ const input = myUploaderInstance.inputElement; // Returns an input element, for
 | `unqueuedFile.fileUploader` | This event is fired after the file was removed from queue.                                                                                  |
 | `error.fileUploader`        | This event is fired when an error occurs when adding files to the queue. A specific error message is also returned together with the event. |
 
+## Implementation notes
+
+### Name attribute ⚠️
```

Comment:
I am missing information that the value returned from this component's inputs will be always in the format of an array.

Reply:
It should be obvious from the example below that the name is then used as an array. Alternatively, @literat please send me a suggestion of what you would add there.

Reply:
I think this is not obvious and should be mentioned that the pair name=value will be always in a format of an array like `name[]=value`.